cypress/integration/translations.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Language Translations
+  User can choose their preffered language
+
+  Background:
+    Given The entry point is the 'Home' page
+    And The user navigates to the 'paymentCode' page
+
+  Scenario: Choose French language
+    Given I am on the 'Payment code' page
```

Comment:
Could potentially use a Scenario Outline for several of these tests, as they're quite similar.